addons/uh60_flir/functions/fnc_perFrame.sqf

```diff
@@ -6,62 +6,16 @@
 
 params ["_vehicle", "_frameTime"];
 
-private _left = ((inputAction "LookLeft") min 1) + ((inputAction "AimLeft") min 1);
-private _right = ((inputAction "LookRight") min 1) + ((inputAction "AimRight") min 1);
-private _up = ((inputAction "LookUp") min 1) + ((inputAction "AimUp") min 1);
-private _down = ((inputAction "LookDown") min 1) + ((inputAction "AimDown") min 1);
-private _wasSlewing = (vtx_uh60_flir_slewX > 0 || vtx_uh60_flir_slewY > 0);
-vtx_uh60_flir_slewX = 0 - _left + _right;
-vtx_uh60_flir_slewY = 0 - _down + _up;
-private _isSlewing = (vtx_uh60_flir_slewX > 0 || vtx_uh60_flir_slewY > 0);
-private _slewingStopped = (_wasSlewing && !_isSlewing);
-private _isDriver = player == driver _vehicle;
-private _isInCamera = ((_isDriver && cameraView == "GUNNER") || !isNil "vtx_uh60_flir_camera");
-
-private _otherCrew = if (_isDriver) then [{
-	private _copilotTurretIndex = [_vehicle] call ace_common_fnc_getTurretCopilot;
-	(_vehicle turretUnit _copilotTurretIndex)
-},{
-	(driver _vehicle)
-}];
-
-vtx_uh60_flir_stabilized = (getPilotCameraTarget _vehicle) # 0;
-// track laser if not slewing yourself and you're the copilot
-if (isLaserOn _vehicle && !_isSlewing && !_isDriver && !_isInCamera) then {
-	_vehicle setPilotCameraTarget (getPosASL (laserTarget _vehicle));
+if (vtx_uh60_flir_setting_animateTurret && {local _vehicle}) then {
+  [_vehicle] call vtx_uh60_flir_fnc_syncAnimation;
```

Comment:
we should consider doing this less often, maybe we run it with a 1 second duration once a second. It'll be convincing enough and it'll cut our network calls on this by a factor 30-60

Reply:
Would it not be the same as having a standard turret?

Reply:
5f7e214